src/Integration.UnitTests/Suppression/SuppressedIssueMatcherTests.cs

```diff
@@ -131,7 +131,9 @@ private IFilterableIssue CreateIssueToMatch(string ruleId, int? startLine, strin
             };
 
         private SonarQubeIssue CreateServerIssue(string ruleId, int? startLine, string lineHash)
-            => new SonarQubeIssue(null, lineHash, startLine, null, null, ruleId, false, flows: null);
+            => new SonarQubeIssue(null, lineHash, null, null, ruleId, false, SonarQubeIssueSeverity.Info,
+                !startLine.HasValue ? null : new IssueTextRange(startLine.Value, 1, 1, 1),
```

Comment:
Returning null textRange if the line is null.

Reply:
Minor: I think it would be easier to read without the negation